Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -228,7 +228,7 @@ namespace AZ::Reflection
                     nullptr);
 
                 const StackEntry& nodeData = m_stack.back();
-                m_serializeContext->EnumerateInstance(&context, nodeData.m_instance, nodeData.m_typeId, nullptr, nullptr);
+                m_serializeContext->EnumerateInstance(&context, nodeData.m_instance, nodeData.m_typeId, nodeData.m_classData, nodeData.m_classElement);
```

Comment:
EnumerateInstance uses the `ClassData` parameter as the class data for the pointer instance supplied.
When `ClassData` is nullptr, it uses the class TypeId to lookup the ClassData from the SerializeContext.

Now the `ClassElement` is used to determine whether the void pointer instance refers to a type T or to a pointer to type T
For example if the type being passed references `AZ::Entity*`, then the void pointer supplied to EnumerateInstance is treated as a `void**` and deferenced in order to access the `AZ::Entity` instance.

Reply:
I'll happily defer to @lumberyard-employee-dm on this one - last I remember we were using a forked copy of this in the edit context, so I'm way out of date on how this bit works.

Reply:
Okay. Based on the discussion above, I think I'll revert it to what it was before since it's not a required change for this functionality to work. We can revisit this later if this becomes an issue.